user/src/main/kotlin/com/sns/user/component/test/repositories/TestUserRepository.kt

```diff
@@ -0,0 +1,29 @@
+package com.sns.user.component.test.repositories
+
+import com.sns.user.component.test.domains.TestUser
+import org.slf4j.LoggerFactory
+import org.springframework.jdbc.core.JdbcTemplate
+import org.springframework.stereotype.Repository
+
+@Repository
+class TestUserRepository(val jdbcTemplate: JdbcTemplate) : ITestUserRepository {
+    private val log = LoggerFactory.getLogger(javaClass)
+
+    override fun save(nickName: String): Int {
+        return jdbcTemplate.update("INSERT INTO test_user (`nick_name`) VALUES('$nickName')")
+    }
+
+    override fun save(user: TestUser): Int {
+        return jdbcTemplate.update("INSERT INTO test_user (`nick_name`) VALUES('${user.nickName}')")
+    }
+
+    override fun findByNickName(nickName: String): TestUser {
+        return jdbcTemplate.queryForObject(
+            "SELECT id, nick_name\n" +
+                "FROM test_user\n" +
+                "WHERE nick_name = '$nickName'\n" +
+                "LIMIT 1",
+            TestUser.MAPPER
```

Comment:
```suggestion
        return jdbcTemplate.queryForObject("""
            SELECT id, nick_name
            FROM test_user
            WHERE nick_name = ?
            LIMIT 1
        """,
            TestUser.MAPPER,
            nickName
```
이런식으로 `+` 를 사용하지 않고 쿼리를 쓸 수 있습니다
추가로 spring jdbc 에서 같은 쿼리가 문자열이면 캐시를 한다고 알고 있어서
? 로 두는게 성능 상으로는 조금 더 나을 수 있어요

웹에서 수정한거여서 indent 가 안 맞을 수도 있어서 확인 부탁드립니다